### DISCUSSION TITLE
FirebaseAuthentication getUser(uid)

### DISCUSSION THREAD

**Discussion by jongbonga** [capawesome-team/capacitor-firebase#547] [Q&A] [answered]
Hi @robingenz is there a way to `FirebaseAuthentication.getUser(uid)` like [here](https://firebase.google.com/docs/auth/admin/manage-users#retrieve_user_data) or it's not something planned?
![firebase google com_docs_auth_admin_manage-users](https://github.com/capawesome-team/capacitor-firebase/assets/12111716/2cfb6351-6d8d-49f5-a732-7df85d7ba537)

**Comment by robingenz** [comment, marked as answer]
This method does only exists in the Firebase Admin SDK. It's not available as client API.

**Reply by jongbonga** [reply]
Ah ! I didn't notice that part, thank you
